### PR TITLE
[merged] Trust show

### PR DESCRIPTION
**Pull request by aweiteka**
```
$ sudo atomic trust show -h
usage: atomic trust show [-h] [-r]

optional arguments:
  -h, --help  show this help message and exit
  -r, --raw   Output policy file as raw json
$ sudo atomic trust show
docker.io                           reject
docker.io/centos                    signed   https://centos.org/sigstore/
private.example.com                 signed
registry.access.redhat.com          signed   https://cdn.redhat.com/containers/signatures
* (default)                         accept
$ sudo atomic trust show --raw
{
    "default": [
        {
            "type": "insecureAcceptAnything"
        }
    ],
    "transports": {
        "dir": {
            "localhost": [
                {
                    "keyType": "GPGKeys",
                    "type": "signedBy",
                    "keyPath": "/home/aweiteka/dev/proj/atomic/tests/unit/fixtures/key2.pub"
                }
            ]
        },
        "docker": {
            "docker.io/centos": [
                {
                    "keyType": "GPGKeys",
                    "type": "signedBy",
                    "keyPath": "/home/aweiteka/dev/proj/atomic/tests/unit/fixtures/key1.pub"
                }
            ],
            "docker.io": [
                {
                    "type": "reject"
                }
            ],
            "registry.access.redhat.com": [
                {
                    "keyType": "GPGKeys",
                    "type": "signedBy",
                    "keyPath": "/home/aweiteka/dev/proj/atomic/tests/unit/fixtures/key2.pub"
                }
            ]
        },
        "atomic": {
            "private.example.com": [
                {
                    "keyType": "GPGKeys",
                    "type": "signedBy",
                    "keyPath": "/home/aweiteka/dev/proj/atomic/tests/unit/fixtures/key1.pub"
                }
            ]
        }
    }
}
```

### CODE REVIEW

**Comment by aweiteka** [comment]
@rhatdan this depends on #599 so I'll rebase after that is merged.

File: Atomic/trust.py [hunk 1]
```diff
@@ -61,6 +62,10 @@ def cli(subparser):
                          choices=['atomic', 'local', 'web'],
                          help=sigstore_help)
     deletep.set_defaults(_class=Trust, func="delete")
+    showp = subparsers.add_parser("show",
+                                  help="Display trust policy for the system")
+    showp.add_argument('-r', '--raw', action='store_true', help="Output policy file as raw json")
```
Comment (rhatdan) [inline Atomic/trust.py]: --json is used in other commands.

File: Atomic/trust.py [hunk 2]
```diff
@@ -334,3 +339,39 @@ def prompt_trust(self, labels):
             if not "y" in confirm.lower():
                 return False
         return True
+
+    def show(self, raw=None):
```
Comment (rhatdan) [inline Atomic/trust.py]: Don't pass in raw, just use self.args.json.

File: Atomic/trust.py [hunk 3]
```diff
@@ -334,3 +339,39 @@ def prompt_trust(self, labels):
...
+        """
+        Display trust policy
+        """
+        if not raw:
+            raw = self.args.raw
+        registry_config_path = util.get_atomic_config_item(["registry_confdir"], self.atomic_config)
+        registry_configs, _ = util.get_registry_configs(registry_config_path)
+        with open(self.policy_filename, 'r+') as policy_file:
```
Comment (rhatdan) [inline Atomic/trust.py]: Do we need to handle the policy_file not existing?

File: Atomic/trust.py [hunk 4]
```diff
@@ -334,3 +339,39 @@ def prompt_trust(self, labels):
...
+            policy = json.load(policy_file)
+            table = {}
+            transport_types = ["docker", "atomic", "web"]
```
Comment (rhatdan) [inline Atomic/trust.py]: Should this become a global of the class?

File: Atomic/trust.py [hunk 5]
```diff
@@ -334,3 +339,39 @@ def prompt_trust(self, labels):
...
+            for tt in transport_types:
+                if tt in policy["transports"]:
```
Comment (rhatdan) [inline Atomic/trust.py]: What happens if "transports" does not exist?

Comment (aweiteka) [inline Atomic/trust.py]: Good catch. Updated to support running show on default policy.

File: Atomic/trust.py (hunk 4 above)
Comment (aweiteka) [inline Atomic/trust.py]: done

File: Atomic/trust.py (hunk 3 above)
Comment (aweiteka) [inline Atomic/trust.py]: done

File: Atomic/trust.py (hunk 2 above)
Comment (aweiteka) [inline Atomic/trust.py]: done

**Comment by rhatdan** [comment]
Could you merge your commits together and then fix the test failures.

**Comment by aweiteka** [comment]
@rhatdan tests are finally passing. Commits squashed.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**